ui/raidboss/data/07-dt/hunts/shaaloani.ts

```diff
@@ -114,30 +118,29 @@ const identifyOrbSafeSpots = (pattern: Point[]) => {
   // 8 covering the rel. NW and SE quadrants with some overlap in the middle
   // since each pattern is composed of a quadrant of AOEs we can check for any orbs in each quadrant
   // (centered around the arena center) and then check the final result to determine the pattern.
-  const detected: boolean[] = [false, false, false, false]; // TL, TR, BL, BR
+  let topRight = false;
+  let topLeft = false;
+  let bottomRight = false;
+  let bottomLeft = false;
   for (const point of pattern) {
     const xGt = point.x > 53.0;
     const yGt = point.y > -825.0;
-    detected[0] = !detected[0] ? !xGt && !yGt : detected[0];
-    detected[1] = !detected[1] ? xGt && !yGt : detected[1];
-    detected[2] = !detected[2] ? !xGt && yGt : detected[2];
-    detected[3] = !detected[3] ? xGt && yGt : detected[3];
+    topLeft = !topLeft ? !xGt && !yGt : topLeft;
+    topRight = !topRight ? xGt && !yGt : topRight;
+    bottomRight = !bottomRight ? !xGt && yGt : bottomRight;
+    bottomLeft = !bottomLeft ? xGt && yGt : bottomLeft;
```

Comment:
I think these might have been swapped?  Looking back at the prior version before this commit, `detected[2]` was bottom left, and `detected[3]` was bottom right, at least according to the comment (and that was working correctly in game when I tested).

Reply:
I find and replaced these but swapped the names it seems...